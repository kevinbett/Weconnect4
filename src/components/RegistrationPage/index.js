import React from 'react';
import Navigation from '../Navigation/index';
import RegisterForm from '../RegistrationForm/RegisterForm';

const RegistrationPage = () => (

   <div>
       <Navigation />
       <RegisterForm />
      
   </div>
);

export default RegistrationPage;
