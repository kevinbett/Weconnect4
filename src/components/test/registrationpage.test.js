import RegistrationPage from '../RegistrationPage/index'
import { shallow } from "enzyme"

describe('Registration page',() => { 
    it ('Registration page renders', () => { 
        expect(shallow(<RegistrationPage/>)).toMatchSnapshot()
    });
} )