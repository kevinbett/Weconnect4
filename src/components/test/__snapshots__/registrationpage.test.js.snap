// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Registration page Registration page renders 1`] = `
<div>
  <Navigation />
  <RegisterForm />
</div>
`;
